library/src/context.cxx

```diff
@@ -0,0 +1,92 @@
+#include "context.h"
+
+#include <vtksys/DynamicLoader.hxx>
+
+namespace f3d
+{
+//----------------------------------------------------------------------------
+context::loader context::glx()
```

Comment:
there is already a f3d::loader, this will be confusing, lets use something else

Reply:
Don't we want to rename the `loader` to `scene` soon?

Reply:
That will be even more confusing for libf3d user updating from f3d 2.5 to f3d 3.0